### DISCUSSION TITLE
1.13 Ocean Flora (Kelp, Seagrass, Sea Pickles, and Coral)

### DISCUSSION THREAD

**Discussion by Rozmir-Rohi** [Roadhog360/Et-Futurum-Requiem#273] [Ideas]
Having the 1.13 ocean flora in 1.7.10 would be really nice.
Pairing this feature with the Mo' Creatures mod would create more vibrant oceans in 1.7.10.

**Comment by happyburger23** [comment]
In combination, the 1.13 swimming (and item) mechanics would be neat to have!
